doc/stdlib/parallel.rst

```diff
@@ -355,18 +331,6 @@ General Parallel Computing Support
    * ``put!`` on a closed channel.
    * ``take!`` and ``fetch`` on an empty, closed channel.
 
-.. function:: RemoteRef()
```

Comment:
You need to change the old signature (whatever after `.. function:: `) to the new signature (which should be the first line in your doc) so that the doc system can find it.